pysat/tests/test_meta.py

```diff
@@ -1067,6 +1162,8 @@ def test_contains_case_insensitive_w_ho(self):
         assert ('NEW21' not in self.meta)
 
     def test_get_variable_name_case_preservation(self):
+        """Test `meta.var_case_name` preserves the case (is case independent?).
+        """
```

Comment:
I may be misunderstanding this test (and those below), but are they for case independence (I still get the same answer regardless of case) or case preservation (the case I enter matters)?

Reply:
Meta is supposed to preserve the case it sees first but respond to inputs insensitively. The var_case_name is the relevant method here. Setting meta up with 'NEW2' means that is what I should get from var_case_name as long as some case variant of 'new2' is provided to it.

It is checking preservation of output and independence of input.